mw-webapp/src/utils/logToConsole.ts

```diff
@@ -0,0 +1,12 @@
+/* eslint-disable no-console */
+
+/**
+ * Return in console {@link textToLog}
+ */
+export const logToConsole = (textToLog: string) => {
+  if (process.env.IS_LOGGER_ENABLED === "true") {
+    console.log(`${textToLog}`);
```

Comment:
```suggestion
    console.log(`${`${textToLog}`}`);
```

Reply:
BTW: it is pretty useful whaen logger could log time (it could help to troubleshoot somitemes)

Reply:
Adde time to log